entity-framework/core/miscellaneous/platforms.md

```diff
@@ -14,18 +16,18 @@ We want EF Core to be available to developers on all modern .NET implementations
 
 The following table provides guidance for each .NET implementation:
 
-EF Core                       | 3.1        | 5.0    | 6.0
------------------------------ | ---------- | ------ | ---
-.NET & .NET Core              | 3.1        | 5.0    | 6.0
-.NET Standard<sup>(1)</sup>   | 2.0        | 2.1    |
-.NET Framework<sup>(2)</sup>  | 4.7.2      |        |
-Mono                          | 5.4        | 6.4    |
-Xamarin.iOS<sup>(3)</sup>     | 10.14      | 12.16  | (use .NET 6)
-Xamarin.Mac<sup>(3)</sup>     | 3.8        | 5.16   | (use .NET 6)
-Xamarin.Android<sup>(3)</sup> | 8.0        | 10.0   | (use .NET 6)
-UWP<sup>(4)</sup>             | 10.0.16299 |        | (use [Windows App SDK](/windows/apps/windows-app-sdk/))
-Unity<sup>(5)</sup>           | 2018.1     | 2021.2 | TBD
-Tizen<sup>(6)</sup>           | 4.0        | 6.0    | (use .NET 6)
+EF Core                       | 6.0 and 7.0
```

Comment:
There may be some value to people in leaving the older versions (e.g. I'm stuck on old X, which version of EF can I use for now, even if unsupported)